app/models/developer.rb

```diff
@@ -24,4 +24,8 @@ def in_pending_tribe?(tribe)
   def inactive?
     status == "inactive"
   end
+
+  def tag_name_list
+    self.tags.all.map { |tag| tag.name }
```

Comment:
Redundant `self` detected.<br>Pass `&:name` as an argument to `map` instead of a block.